tests/test_jobs.py

```diff
@@ -110,17 +110,6 @@ async def foobar(ctx, *args, **kwargs):
     ]
 
 
-async def test_enqueue_job_with_unix_socket(worker, unix_socket_path):
```

Comment:
redislite doesn't work with python 3.12, and getting a socket connection wasn't simple otherwise, hence removing the fixture and this test.